code/modules/surgery/tools.dm

```diff
@@ -86,6 +86,46 @@
 	origin_tech = "materials=1;biotech=1"
 	attack_verb = list("attacked", "slashed", "sawed", "cut")
 
+/obj/item/weapon/circular_saw/energy //For use by medical syndiborgs

```

Comment:
this should be a subtype of the cyborg esword.
it has no real reason to be a circular saw (any item can be made to work in surgery), but it DOES have merits being an esword, the fact you've pasta'd it's code here.